Sources/main.cpp

```diff
@@ -423,8 +408,6 @@ static int SlaveMain(int argc, char **argv) {
   opts.parse(argc, argv);
   HandleSharedOptions(opts);
 
-  ds2::SetLogLevel(ds2::kLogLevelWarning);
```

Comment:
Why are you removing this?

Reply:
Yeah, if we have a log file we want all the debug logs to go there, forcing to warning will suppress these logs, so I think removing this line makes sense.